src/app/ubs/ubs-admin/components/ubs-admin-customers/ubs-admin-customers.component.ts

```diff
@@ -362,13 +364,36 @@ export class UbsAdminCustomersComponent implements OnInit, AfterViewChecked, OnD
     }
   }
 
-  private setColumnWidth(column: any) {
+  private setColumnWidth(column: ColumnParam) {
     const columnEls = Array.from(document.getElementsByClassName('mat-column-' + column.title.key));
     columnEls.forEach((el: any) => {
       el.style.width = column.width + 'px';
     });
   }
 
+  openPopUp(column: ColumnParam, data: string, userId: string): void {
```

Comment:
The method is doing several things: opening the dialog, handling the language selection for the header, updating the table data, and making an API call. We can break this down into smaller methods to improve readability and reusability.